jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-lib = library(identifier: "jenkins@current", retriever: legacySCM(scm))
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
```

Comment:
Where is this library retrieved from?

Reply:
legacySCM means basically local checked out workspace, but it also caches things in-memory in the service instance and i don't know what the reloading rules are. I suspect it asks "has anything changed" to the "localSCM", which is git, or at least it should.